BookingsApi/BookingsApi.Contract/Requests/UpdateHearingAllocationToCsoRequest.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+
+namespace BookingsApi.Contract.Requests
+{
+    public class UpdateHearingAllocationToCsoRequest
+    {
+        /// <summary>
+        ///     List of removed participant Ids
```

Comment:
the description of the property doesn't match. copy and paste?

Reply:
comments rectified